cmd/mattermost/commands/webhook.go

```diff
@@ -262,6 +270,31 @@ func createOutgoingWebhookCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func deleteWebhookCmdF(command *cobra.Command, args []string) error {
+	app, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer app.Shutdown()
+
+	if len(args) < 1 {
+		return errors.New("WebhookID is not specified")
+	}
+
+	webhookId := args[0]
+	errIncomingWebhook := app.DeleteIncomingWebhook(webhookId)
+	errOutgoingWebhook := app.DeleteOutgoingWebhook(webhookId)
+
+	if errIncomingWebhook != nil && errOutgoingWebhook != nil {
+		if errIncomingWebhook != nil {
```

Comment:
This if makes no sense. You will always inter in that if because the `errIncommingWebhook != nil`, because was checked in the previous if. I think this can be deleted with the content.